etc/ui-config/mh_default_org/paella7/default_theme/theme.json

```diff
@@ -0,0 +1,7 @@
+{
+    "styleSheets": [
```

Comment:
The Opencast UIConfigRest.java has a GET  @Path("{component}/{filename}")
That service can't serve a file at @Path("{component}/{theme}/{filename}"). Should all these paella7 themes be copied into the engage-player-paella-7 or some other static hosting module prior to build via maven?

Reply:
Thanks @karendolan for testing this. 
You are right. I tested it with the proxy and did not detect this problem. I thought that everything in that folder was 
served. Sorry.

I will try to fix it!